tests/socket/resource.h

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) eBPF for Windows contributors
+// SPDX-License-Identifier: MIT
+
+#define EBPF_FILE_DESCRIPTION "Socket test"
```

Comment:
```suggestion
#define EBPF_FILE_DESCRIPTION "Socket tests"
```
For consistency with the file name, and also with file descriptions on the other test executables